stories/topics/proc-aws-managing-backups-playbook.adoc

```diff
@@ -0,0 +1,93 @@
+[id="proc-aws-managing-backups-playbook"]
+
+= Managing backups playbook
+
+There is two playbooks to manage your backups, the first one aws_backup_delete which delete one backup and the second one aws_backups_delete which allows to delete multiple backups at once. We will explain the aws_backups_delete in this chapter. The difference is the aws_backups_delete takes a array of string ["backup1","backup2",...] but the aws_backup_delete take only one string which is the name of a specifc backup "backup1"
+
+== Delete backups
```

Comment:
This should be a separate procedure.
Deleting backups

Reply:
I renamed the proc as there is only 1 procedure in that document.